src/pages/components/CertificateForm.js

```diff
@@ -206,7 +206,6 @@ const Form = styled.form`
 `
 
 const TimelineWrapper = styled(Grid.Panel)`
-  max-width: 400px;
```

Comment:
I am not sure if this is important but , this seems to be one of those occasions when something changes unexpectedly elsewhere 

if i go to the certificate view , it now looks like this ( for some reason )

<img width="1385" alt="image" src="https://github.com/digicatapult/sqnc-hyproof-client/assets/58742260/0bc75cf3-f98b-416d-9cfa-88b4cbe33ccc">